microsoft-edge/devtools-guide-chromium/evaluate-performance/index.md

```diff
@@ -99,43 +99,41 @@ When you ran the optimized version of the page, the blue icons move faster.  Why
 
    ![The results of the profile.](../media/evaluate-performance-performance-capture-results.msft.png)
 
-That's an overwhelming amount of data, but it'll all make more sense shortly.
 
 
 <!-- ====================================================================== -->
 ## Analyze the results
 
 Once you have a recording of the page's performance, you can assess the page's performance and find the cause of any performance issues.
 
-1. The **CPU** chart is displayed along the top.  The colors in the **CPU** chart correspond to the colors in the **Summary** panel, at the bottom of the **Performance** tool.  The **CPU** chart shows that these regions make up a large area, meaning that the CPU was maxed out during the recording.  Whenever the CPU is maxed out for long periods, that's an indicator that you should find ways to do less work.
+1. The **CPU** chart is displayed along the top.  The colors in the **CPU** chart correspond to the colors in the **Summary** panel, at the bottom of the **Performance** tool.  The **CPU** chart shows that these regions make up a large area, meaning that the CPU was maxed out during the recording.  Whenever the CPU is maxed out for long periods, that's an indicator that the page is not performing well.
 
    ![The CPU chart and Summary panel.](../media/evaluate-performance-performance-cpu-chart.msft.png)
 
 1. Hover over the **CPU** or **NET** charts.  DevTools shows a screenshot of the page at that point in time.  Move your mouse left and right to replay the recording.  The action is called _scrubbing_, and it's useful for manually analyzing the progression of the performance recording.
 
    ![Hover on a frame.](../media/evaluate-performance-performance-frame-hover.msft.png)
 
-The display indicates that the webpage isn't performing well.  In real scenarios, it may not be so clear whether the page is performing well, so having all of the tools to make measurements comes in handy.
 
 
-#### Bonus: Open the FPS meter
+#### Bonus: Open the Frame Rendering Stats overlay
 
-Another handy tool is the FPS meter, which provides real-time estimates for FPS as the page runs.
+Another handy tool is the **Frame Rendering Stats** overlay, which provides real-time estimates for FPS as the page runs. The **Frame Rendering Stats** overlay is not required for this tutorial but may provide helpful insight.
```

Comment:
```suggestion
Another handy tool is the **Frame Rendering Stats** overlay, which provides real-time estimates for FPS as the page runs. The **Frame Rendering Stats** overlay isn't required for this tutorial, but may provide helpful insight.
```
https://learn.microsoft.com/en-us/style-guide/word-choice/use-contractions
There are some exceptions: when a point needs to be made forcefully, as in technical API Ref, or legal-type pages, sometimes avoid contraction.

Reply:
No strong preference on the contraction. However, I do think the comma should be removed as in the initial commit because what follows it is not an independent clause.

Reply:
I don't have a Resolve Comment button.  Please click Resolve Comment as appropriate on my comment threads.  A thread can always be reopened if needed.